components/edit-collective/sections/CollectiveGoals.js

```diff
@@ -181,6 +181,7 @@ class CollectiveGoals extends React.Component {
           <Box mb={4}>
             <StyledInputField name={this.fields[1].name} label={this.fields[1].label}>
               <StyledSelect
+                inputId="collective-goals"
```

Comment:
Since there can be multiple goals, won't we end up with multiple `input` having the same `id`?